cas-server-support-osf/src/main/java/io/cos/cas/authentication/handler/support/OpenScienceFrameworkPrincipalFromRequestRemoteUserNonInteractiveCredentialsAction.java

```diff
@@ -311,11 +311,13 @@ protected OpenScienceFrameworkCredential constructCredential(final RequestContex
             // https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPAttributeAccess
             final String remoteUser = request.getHeader(REMOTE_USER);
             if (StringUtils.isEmpty(remoteUser)) {
-                logger.error("Invalid Remote User specified as Empty");
+                logger.error("Invalid Remote User Specified as Empty");
```

Comment:
Still need to log the error here in case `notifyRemotePrincipalAuthenticated()` throws an exception first.